src/firebase.js

```diff
@@ -3,7 +3,7 @@ require("firebase/database");
 require("firebase/auth");
 
 var firebaseConfig = {
-  apiKey: "secrets.DBKEY",
+  apiKey: "AIzaSyB4TLJ46tytlPKiT9c0sobcePupnIH22Jo",
```

Comment:
This API key shouldn't be public

Reply:
not my construction site

Reply:
seems like it is my construction site, will be fixed today

Reply:
added environmental variable